server/datastore/mysql/migrations/tables/20250624140757_VPPAppVerifyInstall.go

```diff
@@ -16,7 +16,9 @@ func Up_20250624140757(tx *sql.Tx) error {
 	ALTER TABLE host_vpp_software_installs
 		ADD COLUMN verification_command_uuid VARCHAR(127) NULL,
 		ADD COLUMN verification_at DATETIME(6) NULL,
-		ADD COLUMN verification_failed_at DATETIME(6) NULL
+		ADD COLUMN verification_failed_at DATETIME(6) NULL,
+		ADD INDEX idx_host_vpp_software_installs_verification_at (verification_at),
+		ADD INDEX idx_host_vpp_software_installs_verification_failed_at (verification_failed_at)
```

Comment:
```suggestion
		ADD INDEX idx_host_vpp_software_installs_verification ((verification_at IS NULL AND verification_failed_at IS NULL))
```

I checked our usages in WHERE clauses and the only times we check these columns is together when they're null, plus optionally host ID. I'd throw in partial indexing here to further decrease index size but MySQL can't do that.

I was on the fence on whether to include `host_id` in here, but turning MDM off feels rare enough that we might as well save the writes (and the storage) and narrow things down based on the other functional index instead.